docs/docs/APIdocs/layers/nn/Simple Layers/Clamp.md

```diff
@@ -0,0 +1,52 @@
+## Clamp ##
+
+**Scala:**
+```scala
+val model = Clamp[T](min, max)
+```
+**Python:**
+```python
+model = Clamp(min, max)
+```
+
+A kind of hard tanh activition function with integer min and max
```

Comment:
Are you sure the document is right? https://github.com/torch/nn/blob/master/doc/simple.md#nn.Clamp